samples/XAMLCatalog/XAMLCatalog/UISliderViewController.m

```diff
@@ -58,6 +58,7 @@ - (UITableViewCell*)tableView:(UITableView*)tableView cellForRowAtIndexPath:(NSI
         UILabel* label = [[UILabel alloc] initWithFrame:CGRectMake(xMargin, 0, labelWidth, labelHeight)];
         label.text = @"UISlider (initial = 0, min = 0, max = 1)";
         UISlider* simpleSlider = [[UISlider alloc] initWithFrame:CGRectMake(xMargin, labelHeight, 300, elementHeight)];
+        simpleSlider.accessibilityIdentifier = @"slider_Simple";
```

Comment:
This is used within the external XAMLCatalogTest WinAppDriver test DLL - it's used to identify the slider by its accessibility ID.

---
In reply to: [84973509](https://github.com/Microsoft/WinObjC/pull/1204#discussion_r84973509) [](ancestors = 84973509)